attribution/api_attribution.py

```diff
@@ -112,32 +107,53 @@ def compute_attributions(self, input_text: str, **kwargs):
                 for unit_token_ids in token_ids_per_unit[i_unit + 1 :]
                 for token_id in unit_token_ids
             ]
-            perturbed_input = self.local_tokenizer.decode(
-                left_token_ids + replacement_token_ids + right_token_ids
+            perturbed_input = self.tokenizer.decode(
+                left_token_ids + replacement_token_ids + right_token_ids, skip_special_tokens=True
             )
 
             # Get the output logprobs for the perturbed input
             perturbed_output = self.get_chat_completion(perturbed_input)
 
+
+            if ignore_output_token_location:
+
+                all_top_logprobs = []
+                all_toks = []
+                for ptl in perturbed_output.logprobs.content:
```

Comment:
Does `ptl` stand for `perturbed_top_logprobs`? May want to expand it to make it more explicit